packages/webdriverio/src/utils/findStrategy.ts

```diff
@@ -17,8 +17,8 @@ const XPATH_SELECTOR_REGEXP = [
     /^([a-z0-9|-]*)/,
     // optional . or # + class or id
     /(?:(\.|#)(-?[_a-zA-Z]+[_a-zA-Z0-9-]*))?/,
-    // optional [attribute-name="attribute-selector"]
-    /(?:\[(-?[_a-zA-Z]+[_a-zA-Z0-9-]*)(?:=(?:"|')([a-zA-z0-9\-_. ]+)(?:"|'))?\])?/,
+    // optional [attribute-name="attribute-selector"]. "\p{L}" - all letters (case-insensitive), "\p{N}" - all numbers
+    /(?:\[(-?[_a-zA-Z]+[_a-zA-Z0-9-]*)(?:=(?:"|')([\p{L}\p{N}\-_. ]+)(?:"|'))?\])?/u,
```

Comment:
When using https://regex101.com/ (make sure to select ecmascript) it tells me: `{L} matches a single character in the list {L} (case sensitive)`